src/library.jl

```diff
@@ -901,16 +901,26 @@ $(_thx_clj("distinct"))
 ```jldoctest
 julia> using Transducers
 
-julia> collect(Unique(), [1, 1, 2, 1, 3, 3, 2])
+julia> collect(Unique(), [1, 1, 2, -1, 3, 3, 2])
+3-element Array{Int64,1}:
```

Comment:
```suggestion
4-element Array{Int64,1}:
```

This doctest is failing: https://travis-ci.com/tkf/Transducers.jl/jobs/240572228#L397